client/me/help/controller.js

```diff
@@ -31,6 +31,20 @@ module.exports = {
 		);
 	},
 
+	courses: function( context ) {
+		var CoursesComponent = require( './help-courses' ),
```

Comment:
Can we use ES6 here? :) Also, couldn't we just import `./help-courses` at the top of the file as we do with the other imports?